custom_components/myskoda/climate.py

```diff
@@ -99,6 +100,16 @@ def hvac_action(self) -> HVACAction | None:  # noqa: D102
                 return HVACAction.COOLING
             return HVACAction.OFF
 
+    @property
+    def min_temp(self) -> float:
+        """Return the minimum temperature that can be set."""
+        return 16.0  # Restrict to a minimum of 16°C
```

Comment:
On enyaq minimum is 15.5 (after 16.0 is goes to LO, after 29.5 it goes to HI in the app)